exporter/jaeger/lib/opentelemetry/exporter/jaeger/attributes.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Exporter
+    module Jaeger
+      module Attributes
+        # Identifies Jaeger failed trace exports
+        OTEL_JAEGER_EXPORTER_FAILURE = 'otel.jaeger_exporter.failure'
```

Comment:
IMO the prefixes of these constants can be removed. We're already nested within `OpenTelemetry::Exporter::Jaeger` here, so the context is clear.

These are also not attributes - they're metric names.

I think it'd be clearer to structure this like:
```ruby
module OpenTelemetry
  module Exporter
    module Jaeger
      # Metric counting Jaeger failed trace exports
      EXPORTER_FAILURE = 'otel.jaeger_exporter.failure'

      # Metric recording the time it took to flush Spans to Jaeger
      EXPORTER_REQUEST_DURATION = 'otel.jaeger_exporter.request_duration'
    end
  end
end
```